articles/private-link/private-endpoint-overview.md

```diff
@@ -116,6 +116,7 @@ A private-link resource is the destination target of a specified private endpoin
 | Azure Media Services | Microsoft.Media/mediaservices | keydelivery, liveevent, streamingendpoint |
 | Resource Management Private Links | Microsoft.Authorization/resourceManagementPrivateLinks | ResourceManagement |
 | Azure Databricks | Microsoft.Databricks/workspaces | databricks_ui_api, browser_authentication |
+| Azure Monitor Private Link Scope | Microsoft.insights/privatelinkscopes | azuremonitor |
```

Comment:
```suggestion
| Azure Monitor Private Link Scope | Microsoft.Insights/privatelinkscopes | azuremonitor |
```
@Alfonsoaz01 - should the 'i' be uppercase (as shown in my change)?

Reply:
cc: @asudbring - see question.

Reply:
Yes, Sorry about that, It should